### ISSUE TITLE
Bots have beat this API for the time being, sorry!

### ISSUE THREAD

**Issue by didimaryadi** [hakerdefo/moc-lyrics#3]
how to solve this  === >  "Bots have beat this API for the time being, sorry!" 

is it no longer running?

**Comment by hakerdefo** [comment]
@didimaryadi 

makeitpersonal has stopped their service that's why you are seeing that message. There are a few things that can be done and I've implemented them in `cmus-lyrics`. I think they can very easily be adapted into `moc-lyrics`. I'll do it but I can't promise you an exact date but I can encourage you to take a look at `cmus` and `cmus-lyrics` in the meantime.

https://github.com/hakerdefo/cmus-lyrics

https://cmus.github.io/

**Comment by didimaryadi** [comment]
yes makeitpersonal has stopped, I replaced it using "lyrics" on github then I combined it into your script :-D
because I've used the moc and it's comfortable.  

![Capture](https://github.com/hakerdefo/moc-lyrics/assets/106165017/1f223a13-7fdd-4942-9d21-fd168cea6e57)
![Capture1](https://github.com/hakerdefo/moc-lyrics/assets/106165017/c11abecf-a461-46b6-b986-5317e0f910ec)

**Comment by hakerdefo** [comment]
Great work @didimaryadi :+1:
